build/bin/network-validator.go

```diff
@@ -43,9 +49,8 @@ func (c *reachabilityConfig) LoadFromYaml(filePath string) error {
 }
 
 type endpoint struct {
-	Host        string `yaml:"host"`
-	Ports       []int  `yaml:"ports"`
-	TLSDisabled bool   `yaml:"tlsDisabled"`
+	Host  string `yaml:"host"`
+	Ports []int  `yaml:"ports"`
```

Comment:
The `tlsDisabled` here is marshalled from the config.yaml file, and shouldn't be removed. Its used for some urls that have self-signed certs or other reason to disable tls checks.

Reply:
ack, adding it back, I thought we could remove it as there's no active use 
> tlsDisabled migrated to flag (--no-tls) as there's no usage in yaml, but it should be configurable thru remote calls

but I think we'd still need a flag for _tls disabling_ (like _--no-tls_) (mb for each endpoint) otherwise the caller can't do that without file modification which costs us a deployment... (as we change the file in docker)

Reply:
Currently, there is one config using this field in the yaml. I agree the option should still be there via flag, it should just set tlsDisabled for all endpoints

Reply:
_TLSDisabled_ added back